setup.py

```diff
@@ -66,7 +66,8 @@
     entry_points=entry_points,
     install_requires=[
         'astropy>=1.1',
-        'numpy'
+        'numpy',
+        'bs4'
```

Comment:
What about `requests`?

Reply:
I replaced `requests` with `urllib`

Reply:
Oh, ok. Yes, I remember reading about that earlier. Sorry for the noise!

Reply:
Hmm... Are you sure it is not `'beautifulsoup4'` as per PyPI listing? https://pypi.org/project/beautifulsoup4/

Reply:
```
$ conda list soup
# Name                    Version                   Build  Channel
beautifulsoup4            4.6.3                     <pip>
```

Reply:
Either works. The author has a dummy package called `bs4` that is aliased to `beautifulsoup4`. For some reason they decided to call it `beautifulsoup4`, yet if you try to import a package by that name, it will raises an import error. It can only be imported with `import bs4`, regardless of the route of installation (i.e. `pip install bs4` or `pip install beautifulsoup4`). /shrug

Reply:
@jhunkeler said best to make it something that `conda` also recognize.